sass/animate/_times.scss

```diff
@@ -82,5 +88,5 @@ $times: () !default;
 @mixin add-times(
   $maps...
 ) {
-  $times: _a_map-multimerge($times, $maps...) !global;
-}
+  $times: utils.multi-merge($times, $maps...) !global;
+}
```

Comment:
Can you set up your text editor to keep an end-of-file newline on save? I think that's a setting in vscode.